editor/src/clj/editor/game_object.clj

```diff
@@ -481,8 +481,9 @@
                                          (contains? tags :embeddable))
                                  ext))
                              (workspace/get-resource-type-map workspace))]
-    (when-let [resource (first (resource-dialog/make workspace project {:ext component-exts :title "Select Component File"}))]
-      (add-referenced-component! go-id resource select-fn))))
+    (when-let [resources (resource-dialog/make workspace project {:ext component-exts :title "Select Component File" :selection :multiple})]
+      (doseq [resource (if (map? resources) (vals resources) resources)]
+        (add-referenced-component! go-id resource select-fn)))))
```

Comment:
It feels strange we didn't do it before.
Is it because in such case each adding will be a separate action and we will undo it one by one? Is it really a problem?

Reply:
I think it may just be that we added the ability to select multiple items in the dialog after this code was written. 🤔